WIP Cloud Costs + Free Trial pages, Standard > Plan Your Cluster rewrite (#18883)
cockroachdb/docs: src/current/cockroachcloud/costs.md

# CockroachDB Cloud Costs \[WIP\]

> The cost structure for a CockroachDB Cloud organization depends on its deployment types, which correspond to CockroachDB Cloud tiers.

Reviewer: ```suggestion
The cost structure for a CockroachDB Cloud organization depends on its plan.
```

> ## Cost structures across CockroachDB Cloud tiers

Reviewer: ```suggestion
## Cost structures across CockroachDB Cloud plans
```
